### PR TITLE
Feature/sml/ui polish

### PR DESCRIPTION
**Pull request by lederer**
UX cleanup and cross-browser sweep.

- Isochrone slider was rendering terribly on IE and Edge. Fixed that. #778.
- Some nasty layout problems on IE and Edge. Fixed those. #779.
- Fixed the weird gap between sidebar and map with sidebar contents scroll. #752.
- Added a (dummy) "Go" button to mobile itineraries to strongly indicate what to do next. #775.

### CODE REVIEW

File: src/app/scripts/cac/map/cac-map-templates.js [hunk 1]
```diff
@@ -243,71 +244,73 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
...
+                    '</div>',
+                '{{/each}}',
+                '<div class="directions-leg directions-leg-destination">',
+                    '<div class="directions-step directions-step-destination">',
+                        '<div class="directions-instruction">Arrive {{data.end.text}}</div>',
+                        '<div class="directions-time">at {{datetime data.end.time}}</div>',
+                    '</div>',
                 '</div>',
-            '</div>'
+            '</div>',
```
Comment (flibbertigibbet) [inline src/app/scripts/cac/map/cac-map-templates.js]: Drop trailing comma

File: src/app/scripts/cac/map/cac-map-templates.js [hunk 2]
```diff
@@ -232,6 +232,7 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
                     '</div>',
                 '</div>',
             '</div>',
+            '<div class="route-summary-go-btn">Go</div>',
```
Comment (flibbertigibbet) [inline src/app/scripts/cac/map/cac-map-templates.js]: This isn't visible or wired to do anything. Why did this `div` get added?

Comment (lederer) [inline src/app/scripts/cac/map/cac-map-templates.js]: It's visible in mobile only. Client feedback indicated it's not entirely clear you have to tap the itinerary item at bottom of screen to open it. This is a dummy button, of sorts, to make that clear.

Comment (flibbertigibbet) [inline src/app/scripts/cac/map/cac-map-templates.js]: Okay, gotcha.

**Pull request merged**